javalin/pom.xml

```diff
@@ -132,7 +138,6 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <scope>test</scope>
```

Comment:
I was getting gson syntax exception class def not found

so needed to include gson as a dependency - but saw it was already added so I removed the test scope

I can move it to the right place in the pom file if this is fine